.circleci/config.yml

```diff
@@ -547,45 +405,109 @@ workflows:
-      - "7.3 zts 20180731-zts": *BUILD_PACKAGE_FORKFLOW
+      - "PHP 56 20131106": *BUILD_PACKAGE_FORKFLOW
+      - "PHP 70 20151012": *BUILD_PACKAGE_FORKFLOW
+      - "PHP 71 20160303": *BUILD_PACKAGE_FORKFLOW
+      - "PHP 72 20170718": *BUILD_PACKAGE_FORKFLOW
+      - "PHP 73 20180731": *BUILD_PACKAGE_FORKFLOW
+      - "PHP 56 zts 20131106-zts": *BUILD_PACKAGE_FORKFLOW
+      - "PHP 70 zts 20151012-zts": *BUILD_PACKAGE_FORKFLOW
+      - "PHP 71 zts 20160303-zts": *BUILD_PACKAGE_FORKFLOW
+      - "PHP 72 zts 20170718-zts": *BUILD_PACKAGE_FORKFLOW
+      - "PHP 73 zts 20180731-zts": *BUILD_PACKAGE_FORKFLOW
       - "package extension":
           requires:
-            - "5.4 20100412"
-            - "5.6 20131106"
-            - "7.0 20151012"
-            - "7.1 20160303"
-            - "7.2 20170718"
-            - "7.3 20180731"
-            - "5.6 zts 20131106-zts"
-            - "7.0 zts 20151012-zts"
-            - "7.1 zts 20160303-zts"
-            - "7.2 zts 20170718-zts"
-            - "7.3 zts 20180731-zts"
+            - "PHP 54 20100412"
+            - "PHP 56 20131106"
+            - "PHP 70 20151012"
+            - "PHP 71 20160303"
+            - "PHP 72 20170718"
+            - "PHP 73 20180731"
+            - "PHP 56 zts 20131106-zts"
+            - "PHP 70 zts 20151012-zts"
+            - "PHP 71 zts 20160303-zts"
+            - "PHP 72 zts 20170718-zts"
+            - "PHP 73 zts 20180731-zts"
       - "package verification":
           requires:
             - "package extension"
   build:
     jobs:
+      - unit_tests:
+          name: "Unit tests 54"
+          image_tag: ddtrace_alpine_php-5.4-debug
+      - unit_tests:
+          name: "Unit tests 56"
+          image_tag: ddtrace_alpine_php-5.6-debug
+      - unit_tests:
+          name: "Unit tests 70"
+          image_tag: ddtrace_alpine_php-7.0-debug
+      - unit_tests:
+          name: "Unit tests 71"
+          image_tag: ddtrace_alpine_php-7.1-debug
+      - unit_tests:
+          name: "Unit tests 72"
+          image_tag: ddtrace_alpine_php-7.2-debug
+      # - unit_tests:
+      #     name: "Unit tests 70-zts"
+      #     image_tag: ddtrace_alpine_php-7.0-zts-debug # zts build is failing atm due to 2 memory leaks
+      # - unit_tests:
+      #     name: "Unit tests 73"
+      #     version: "7.3" # TODO: unit test for 7.3 are failing due to memory leaks - 7.3 not yet officially supported
+      - integration_tests:
+          name: "Intg tests 54"
+          image_tag: "ddtrace_alpine_php-5.4-debug"
+          integration_testsuite: "test-integrations-54"
+      - integration_tests:
+          name: "Intg tests 56"
+          integration_testsuite: "test-integrations-56"
+          image_tag: "ddtrace_alpine_php-5.6-debug"
+      - integration_tests:
+          name: "Intg tests 70"
+          integration_testsuite: "test-integrations-70"
+          image_tag: "ddtrace_alpine_php-7.0-debug"
+      - integration_tests:
+          name: "Intg tests 71"
```

Comment:
Please don't hate me 😞 what about `s/Intg/Integrations/g`

Reply:
I agree with all my :heart: but  
I shortened it to make PHP version actually visible in Circle Ci UI. Which makes long names essentially useless

Reply:
![image](https://user-images.githubusercontent.com/812891/54024136-2483df00-4197-11e9-82d8-7b9ff010b9cb.png)

Reply:
See `Static Analy...`
These changes actually started with `Integration tests 54` and even `Web integration tests 54` etc. 
But this turned into a bunch of 
`Integration tes...`
`Integration tes...`
`Integration tes...`
`Integration tes...`
`Integration tes...`
`Web integratio...`
`Web integratio...`
`Web integratio...`
`Web integratio...`
`Web integratio...`

I would have start with with version number but. The name of a job cannot start with a number in `2.1`.

Maybe something like `PHP 54 Integration tests` could work WDYT ?

Reply:
Hmm actually I like that more than current short forms. Let me quickly change.

Reply:
WDYT? 
![image](https://user-images.githubusercontent.com/812891/54026266-8e06ec00-419d-11e9-9b8b-1d3bffa182b1.png)

Reply:
I sort of hate that I asked you to change that 😄! I am ok with this, but feel free to go back to the initial proposal or to abbreviate Integrations again if you prefer 👍

Reply:
You were right, I didn't like them either. But with naming I just usually try to use >first< thing that came to my mind,  turns out 2nd idea was much better :D